### DISCUSSION TITLE
confirm module settings for Splatter (works for PCs, but not for npcs)

### DISCUSSION THREAD

**Discussion by wmve** [Rughalt/D35E#763] [Q&A] [answered]
the module splatter causes blood splatter on the map according to damage suffered by a token (can even create blood trails).  however, i've only seen this work on PCs.  (i've thought i'd had this working on NPCs at one point, but i can't seem to reproduce that now).

![image](https://user-images.githubusercontent.com/55966777/132755183-55d4e89a-c013-4efd-98c4-05748967f7d7.png)

these are the settings i'm using.  the last four are defaults, i believe.  i've only messed with the sliders.  i'm going to guess this was designed with 5e in mind, which often leaves us in 3.5e land on the short end of the stick.

this isn't a priority for me, but this is a really cool module and i'd love to see this work on npcs/monsters.  but since this was discussed in #development on the discord server, i'm logging a discussion here to eventually capture some kind of an answer from someone.

**Comment by michalgm** [comment, marked as answer]
creature type should be 'data.attributes.creatureType' - not sure if there's a separate 'custom' type - i just set both to that

**Reply by Rughalt** [reply]
This is the correct place, and after system update to pull the type from racial HD it will correctly populate.
